boot/initramfs20dirs.go

```diff
@@ -47,21 +47,28 @@ var (
 	InitramfsUbuntuSeedDir string
 
 	// InitramfsWritableDir is the location of the writable partition during the
-	// initramfs.
+	// initramfs. Note that this may refer to a temporary filesystem or a
+	// physical partition depending on what operating mode the system is in.
```

Comment:
we stopped calling these operating mode but use system mode.  here probably just s/operating//

Reply:
ah damn you're right I forgot it got renamed again, will change this